.github/workflows/ci-cd-pull-request-release-please.yml

```diff
@@ -72,6 +72,10 @@ jobs:
     needs: [dry-run-deploy-infra-staging, dry-run-deploy-apps-staging]
     runs-on: ubuntu-latest
     steps:
+      - uses: step-security/harden-runner@c6295a65d1254861815972266d5933fd6e532bdf # v2.11.1
```

Comment:
Separate into a composite action like "build-schema"? So we don't need to pin version everywhere we use this